pkg/config/types.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+const (
+	// counter - attacher package
+	CPUCycle       = "cpu_cycles"
```

Comment:
The names of the cgroup metrics start with `cgroup`, which makes the origin of the metric very transparent.

So, could we add the `kubelet_` prefix to the kubelet metrics and maybe the `hardware_counter_` prefix to its metrics too?
Note that, I prefer hardware_counter_ instead of bpf_ because if we change the method to collect the hardware counter, e.g., using perf, the prefix name will still make sense....

On the other hand, cpu time is specific to bpf, so I think the metric source prefix should be `bpf_`. Note that, I think this metric will be deprecated later since cgroup also has the cpu time...

Reply:
@marceloamaral  
Changing the metric name affects the other components ( e.g., online trainer of kepler-model-server ) and also has a different purpose from this PR. 
Let's make renaming list to a single note and do the metric renaming in the different PR.

Reply:
Ok, sounds good!